client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
Hehe, in 2., you can add this condition:

`if (defaultAccountId.length && acc.id === defaultAccountId)`